e2e-tests/tests/init-deploy/00-deploy-operator.yaml

```diff
@@ -8,12 +8,28 @@ spec:
     requests.memory: "10Gi"
     limits.cpu: "10"
     limits.memory: "12Gi"
+commands:
+  - script: |-
+      null
+      ---
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+      init_temp_dir # do this only in the first TestStep
+
+      deploy_operator
+      deploy_non_tls_cluster_secrets
+      deploy_tls_cluster_secrets
+      deploy_client
 ---
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 timeout: 10
 commands:
   - script: |-
+      null
+      ---
```

Comment:
is this intentional?

Reply:
No it is not, seems that running `make kuttl-shfmt` generates it. Let me revert this.